### PR TITLE
add send_templated_email

### PR DESCRIPTION
**Pull request by xfumihiro**

### CODE REVIEW

File: lib/ex_aws/ses.ex [hunk 1]
```diff
@@ -108,6 +108,38 @@ defmodule ExAws.SES do
     request(:send_raw_email, params)
   end
 
+  @doc """
+  Send a templated Email.
+  """
+  @type send_templated_email_opt :: {:configuration_set_name, String.t}
+    | {:from_arn, String.t}
```
Comment (lessless) [inline lib/ex_aws/ses.ex]: I don't see `from_arn` option at https://docs.aws.amazon.com/ses/latest/APIReference/API_SendTemplatedEmail.html

**Review by lessless — CHANGES_REQUESTED** [review]
Hi @xfumihiro! 

There are few things here - `from_arn` doesn't seems to be an options that SES can recognize and second I want to ask to add test coverage for the `send_templated_email_opt`.

Thanks for the PR!

**Comment by xfumihiro** [comment]
done!

File: test/lib/ses_test.exs [hunk 2]
```diff
@@ -86,6 +86,56 @@ defmodule ExAws.SESTest do
...
+        "ReturnPath" => "feedback@example.com",
+        "ReturnPathArn" => "arn:aws:ses:us-east-1:123456789012:identity/example.com",
+        "Source" => "user@example.com",
+        "SourceArn" => "east-1:123456789012:identity/example.com",
+        "Tags.member.1.Name" => "tag1", "Tags.member.1.Value" => "tag1value1",
+        "Tags.member.2.Name" => "tag2", "Tags.member.2.Value" => "tag2value1",
+        "Template" => "my_template", "TemplateData" => template_data
+      }
+
+      assert expected == ExAws.SES.send_templated_email(
```
Comment (samhstn) [inline test/lib/ses_test.exs]: `ExAws.SES` -> `SES` (it's aliased in this module)

**Comment by lessless** [comment]
Great stuff @xfumihiro ! One last thing before merging it in is a parser test - https://github.com/ex-aws/ex_aws_ses/blob/master/test/lib/ses/parser_test.exs :)

**Review by lessless — CHANGES_REQUESTED** [review]
Great stuff @xfumihiro ! One last thing before merging it in is a parser test - https://github.com/ex-aws/ex_aws_ses/blob/master/test/lib/ses/parser_test.exs :)

**Comment by xfumihiro** [comment]
done!

**Review by lessless — APPROVED** [review]
Great! Thanks for contribution ❤️ !

**Review by lessless — APPROVED** [review]

**Comment by lessless** [comment]
@xfumihiro available in 2.0.2 https://hex.pm/packages/ex_aws_ses/2.0.2

**Pull request merged**